src/daemon/server.py

```diff
@@ -694,7 +707,7 @@ def launch_service(root_path, service_command):
         startupinfo.dwFlags |= subprocess.STARTF_USESHOWWINDOW  # type: ignore
 
     # CREATE_NEW_PROCESS_GROUP allows graceful shutdown on windows, by CTRL_BREAK_EVENT signal
-    if platform == "win32" or platform == "cygwin":
+    if sys.platform == "win32" or sys.platform == "cygwin":
```

Comment:
This change to use `sys.platform` is necesary to make mypy understand that `subprocess.CREATE_NEW_PROCESS_GROUP` exists (because it doesn't exists depending on the operating system).